rootfs/rootfs/etc/rc.d/vmtoolsd

```diff
@@ -0,0 +1,3 @@
+#!/bin/sh
+
+/usr/local/bin/vmtoolsd --background /var/run/vmtoolsd.pid
```

Comment:
It is a good citizen :smile: added a comment to note that, just tested with VirtualBox:

![vmtoolsd in virtualbox](https://www.dropbox.com/s/rvzqtdo18r7mp15/Screenshot%202015-02-26%2015.00.39.png?dl=1)